openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/workflows/dbt.md

```diff
@@ -0,0 +1,308 @@
+# Metadata
+
+dbt Pipeline Configuration.
+## Properties
+
+$$section
+### dbt Config Source $(id="dbtConfigSource")
+
+You can choose one if the 5 sources to fetch the dbt files:
+1. **dbt Local Config**: Config to fetch dbt files from local system.
+2. **dbt HTTP Config**: Config to fetch dbt files from http or file server.
+2. **dbt Cloud Config**: Config to fetch the dbt files from dbt cloud APIs
+2. **dbt S3 Config**: Config to fetch the dbt files from s3.
+2. **dbt GCS Config**: Config to fetch the dbt files from gcs.
+$$
+
+$$section
+###dbt Local Config $(id="dbtLocalConfig")
+
+In this configuration we will be fetching the dbt `manifest.json`, `catalog.json` and `run_results.json` files from the local system or in the OpenMetadata container.
+$$
+
+$$section
+#### dbt Catalog File Path $(id="dbtCatalogFilePath")
+
+Provide the full file path of the dbt catalog.json file. It will be of the type `/root/folder/catalog.json`.
+This is an optional file, if not provided catalog.json metadata will not be imported.
+$$
+
+$$section
+#### dbt Manifest File Path $(id="dbtManifestFilePath")
+
+Provide the full file path of the dbt manifest.json file. It will be of the type `/root/folder/manifest.json`.
+This is a mandatory file for dbt ingestion.
+$$
+
+$$section
+#### dbt Run Results File Path $(id="dbtRunResultsFilePath")
+
+Provide the full file path of the dbt run_results.json file. It will be of the type `/root/folder/run_results.json`.
+This is an optional file, if not provided dbt tests and test results will not be imported.
+$$
+
+************************************************************************************************************************
+
+$$section
+###dbt HTTP Config $(id="dbtHttpConfig")
+
+In this configuration we will be fetching the dbt `manifest.json`, `catalog.json` and `run_results.json` files from http or file server.
+$$
```

Comment:
Can we add something here about this being the method to use if your files are stored in version control tool (e.g. Github, Bitbucket, etc.)